Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -885,9 +885,15 @@ inline TextUnderlineOffset BuilderConverter::convertTextUnderlineOffset(BuilderS
     case CSSValueAuto:
         return TextUnderlineOffset::createWithAuto();
     default:
+        if (primitiveValue.isPercentage())
```

Comment:
I think this entire converter can be simplified to:

```c++
auto length = downcast<CSSPrimitiveValue>(value).convertToLength<FixedIntegerConversion | FixedFloatConversion | PercentConversion | CalculatedConversion | AutoConversion>(builderState.cssToLengthConversionData());
return TextUnderlineOffset::createWithLength(WTFMove(length));
```

Reply:
thank you!